src/widgets/merch/ui/merch.module.scss

```diff
@@ -10,10 +10,11 @@
     }
   }
 
-  .picture {
+  .image {
     width: 680px;
     height: 700px;
     margin: 0 auto;
+    object-fit: cover;
 
     @include media-laptop {
```

Comment:
I'm not sure here, but we can avoid using `margin-top: 24px;` and `margin: 0 auto;` property `margin: 0 auto;` - looks redundant because the properties from the `.column-2` class center the picture. and the `margin-top: 24px;` - property can be replaced by increasing the gap size on the media-laptop size

Reply:
Done